src/Core/Grid/DataProvider/DoctrineGridDataProvider.php

```diff
@@ -61,6 +78,12 @@ public function getData(SearchCriteriaInterface $searchCriteria)
         $records = $searchQueryBuilder->execute()->fetchAll();
         $recordsTotal = (int) $countQueryBuilder->execute()->fetch(PDO::FETCH_COLUMN);
 
+        $this->hookDispatcher->dispatchForParameters('modifyGridQueryBuilder', [
```

Comment:
here, how can you manage to change the grid query builder for a specific Grid? :)

Reply:
you know how you can update this hook name ^_^ => https://github.com/PrestaShop/docs/pull/116

Reply:
i guess i should remove it as its done in our PR?

Reply:
yes :)

Reply:
ill update it tomorrow morning, is there anything else to do regarding grid? tests, docs?

Reply:
hmm ... i've just merged https://github.com/PrestaShop/docs/pull/116, do you think you can start to document every column type for instance?

I'll update architecture and docs for architecture soon (after merge of  https://github.com/PrestaShop/PrestaShop/pull/9470)

Reply:
and then https://github.com/PrestaShop/PrestaShop/pull/9402 :D 

Ok, so ill work on Columns type docs, i also have some docs regarding DataFactory, so ill see if i can update it. :)